### PR TITLE
Docs fixes

### PR DESCRIPTION
**Pull request by amoodie**
This adds checks to the docs during building, so we can begin to enforce valid documentation on pull requests. 

Other than that, I fixed a few small bugs in the docs, and added a basic usage example to the 10min tutorial.

### CODE REVIEW

File: docs/source/guides/10min.rst [hunk 1]
```diff
@@ -3,3 +3,36 @@
...
+.. code::
+
+    >>> for _t in range(0, 1):
+    >>>     delta.update()
+
+    >>> delta.finalize()
+
+That's it! You ran the pyDeltaRCM model for one timestep. 
+
+We can visualize the delta bed elevation, thought it's not very exciting after only one timestep...
```
Comment (elbeejay) [inline docs/source/guides/10min.rst]: Typo, "thought" should be "though"

**Review by elbeejay — COMMENTED** [review]
1 typo in the 10-minute guide. Otherwise all looks good to me.

File: docs/source/guides/10min.rst (hunk 1 above)
Comment (amoodie) [reply docs/source/guides/10min.rst]: thanks, fixed.

Comment (amoodie) [reply docs/source/guides/10min.rst]: will merge after checks pass

**Review by elbeejay — APPROVED** [review]

**Pull request merged**